app/server/src/generator/templates/template2/index.js

```diff
@@ -32,12 +32,9 @@ function generateProfileSection(profile) {
 
     if (names.length === 1) {
       nameStart = names[0]
-    } else if (names.length === 2) {
-      nameStart = names[0]
-      nameEnd = names[1]
-    } else {
-      nameStart = names.slice(0, names.length - 1).join(' ')
-      nameEnd = names[names.length - 1]
+    }  else {
+	  nameStart = names[0]
+      nameEnd = names.slice(1,names.length).join(' ')
     }
```

Comment:
Indentation is a bit messed up here, and I believe it might be because you're using a `tab` somewhere. The project is using 2-space indent.